OpenEXR/IlmImf/ImfFastHuf.cpp

```diff
@@ -153,7 +153,7 @@ FastHufDecoder::FastHufDecoder
             int runLen = readBits (8, currBits, currBitCount, currByte) +
                          SHORTEST_LONG_RUN;
 
-            if (symbol + runLen > maxSymbol + 1)
+            if (symbol + runLen > Int64(maxSymbol + 1))
```

Comment:
Is there a reason to sometimes use `static_cast<Int64>` and sometimes `Int64` as a casting operator?

Reply:
We recently settled on the use of static_cast<>() instead of cast operators going forward, since it more clearly communicates that something intentional is taking place.  Described in the coding style section here: https://github.com/openexr/openexr/blob/master/CONTRIBUTING.md. We'll update existing code as it comes up.

Reply:
I've converted all the constructor notation to static_cast